rest/server_context.go

```diff
@@ -985,6 +985,10 @@ func (sc *ServerContext) logStats() error {
 
 	AddGoRuntimeStats()
 
+	if err := sc.statsContext.addNetworkInterfaceStatsForHostnamePort(*sc.config.Interface); err != nil {
+		base.Warnf(base.KeyAll, "Error getting network interface resource stats: %v", err)
+	}
```

Comment:
Good point.  I will change this from:

```
	StatKeyNetworkInterfaceBytesSent    = "net_bytes_sent"
	StatKeyNetworkInterfaceBytesRecv    = "net_bytes_recv"
```

to:

```
	StatKeyPubNetworkInterfaceBytesSent    = "pub_net_bytes_sent"
	StatKeyPubNetworkInterfaceBytesRecv    = "pub_net_bytes_recv"
	StatKeyAdminNetworkInterfaceBytesSent    = "admin_net_bytes_sent"
	StatKeyAdminNetworkInterfaceBytesRecv    = "admin_net_bytes_recv"
```

that seems like it would provide more value than aggregating into a single stat.

Reply:
I pushed up a change for this in b36217c4425283d58aacfc3fc92dbe7fd81a68a1